.github/workflows/pythonpackage.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9]
+        python-version: ["3.10"]
```

Comment:
I disagree that we should require 3.10

Reply:
Already an requirement in ISAR and needed for running the test

Reply:
I see